packages/e2e-tests/specs/editor/various/writing-flow.test.js

```diff
@@ -216,6 +216,7 @@ describe( 'Writing Flow', () => {
 		await page.keyboard.up( 'Shift' );
 		await pressKeyWithModifier( 'primary', 'i' );
 		await page.keyboard.press( 'ArrowLeft' );
+		await page.keyboard.press( 'ArrowLeft' );
```

Comment:
This makes sense. First you need to navigate out of italics then out of bold. The old behaviour doesn't make sense.